src/main/java/ag/boersego/bgjs/V8TextureView.java

```diff
@@ -324,6 +324,12 @@ private void sendTouchEvent(String type) {
         touchEventObj.setV8Field("touches", touches);
 
         mBGJSGLView.onEvent(touchEventObj);
+
+        for (final JNIV8GenericObject touch : touchObjs) {
+        	touch.dispose();
+		}
```

Comment:
looks like wrong indentation ;-)

Reply:
There's a lot of old tab indentation. It looks alright in the IDE with the correct tab size. Maybe we should do a separate tabs-to-spaces sweep commit.